app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -344,25 +348,26 @@ protected void updateCardViewImage(@Nullable String url, @Nullable String blurHa
             try {
                 if (url == null) {
                     Glide.with(mCardView.getContext())
-                            .load(mDefaultCardImage)
-                            .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
-                            .into(mCardView.getMainImageView());
+                        .load(mDefaultCardImage)
+                        .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
+                        .into(mCardView.getMainImageView());
                 } else {
-                    GlideUtils.blurHashPlaceholder(
+                    BlurHashUtils.createBlurHashDrawable(
+                        lifecycleOwner,
+                        blurHash,
+                        (aspect > 1) ? (int) Math.round(32 * aspect) : 32,
+                        (aspect >= 1) ? 32 : (int) Math.round(32 / aspect),
+                        bitmap -> {
```

Comment:
When a result is returned (nullable, always called once) we start loading with glide, the bitmap is wrapped in a bitmapdrawable here instead of in the function.